crates/pipeline_manager/src/db.rs

```diff
@@ -115,7 +115,16 @@ impl ProjectStatus {
             Some("success") => Ok(Self::Success),
             Some("pending") => Ok(Self::Pending),
             Some("compiling") => Ok(Self::Compiling),
-            Some("sql_error") => Ok(Self::SqlError(error_string.unwrap_or_default())),
+            Some("sql_error") => {
+                let error = error_string.unwrap_or_default();
+                if let Ok(messages) = serde_json::from_str(&error) {
+                    Ok(Self::SqlError(messages))
+                }
+                else {
+                    log::debug!("Expected valid json for SqlCompilerMessage but got {:?}, did you update struct without adjusting the database?", error);
```

Comment:
This could be `log::warn` or even `log::error`.  Also, for consistency with the rest of the code base, let's import `log::error` in module preamble.